api/services/reviewService.js

```diff
@@ -0,0 +1,52 @@
+const { reviewDao } = require("../models");
+const { commentValidate } = require("../utils/validate-check");
+const { reviewValidate } = require("../utils/validate-check");
+const { userMovieValidate } = require("../utils/validate-check");
+const jwt = require("jsonwebtoken");
+
+const createReview = async (accessToken, rating, comment, movieId) => {
+  if (!rating || !comment || !movieId) {
+    const error = new Error("KEY_ERROR");
+    error.statusCode = 400;
+    throw error;
+  }
+
+  if (rating < 1 || rating > 5) {
+    const error = new Error("INVALID_RATING");
+    error.statusCode = 400;
+    throw error;
+  }
+
+  commentValidate(comment);
```

Comment:
```suggestion
  await commentValidate(comment);
```
확실한 동기/비동기 처리를 위해 await를 활용해주세요.